contrib/ceph-build-config.sh

```diff
@@ -0,0 +1,530 @@
+#!/usr/bin/env bash
+set -uo pipefail
+
+# This is mostly an internal representation of 'FLAVORS_TO_BUILD'.
+# These build scripts don't need to have the aarch64 part of the distro specified
+# I.e., specifying 'luminous,centos-arm64,7' is not necessary for aarch64 builds; these scripts
+#       will do the right build. See configurable CENTOS_AARCH64_FLAVOR_DISTRO below
+X86_64_FLAVORS_TO_BUILD="kraken,centos,7 luminous,centos,7 mimic,centos,7"
```

Comment:
I'm not very fan of redefining this while the Makefile already have the supported image in FLAVORS

Reply:
The makefile doesn't distinguish between x86_64 and aarch64 supported images. Additionally, these release scripts won't work for openSUSE. So they must be redefined.